app/models/team.rb

```diff
@@ -4,10 +4,14 @@
 #
 #  id         :integer          not null, primary key
 #  name       :string
+#  description:text
+#  kindofsport:string
+#  private    :boolean
 #  created_at :datetime         not null
 #  updated_at :datetime         not null
 #
 
 class Team < ApplicationRecord
   validates :name, presence: true
```

Comment:
Maybe the `private` field should be mandatory as well – to my mind, the checkbox is always either checked or not checked thus there should always be a value for the boolean `private` field when using the frontend view. By checking the `private` field's presence in the Team class, we could also force API users other than the frontend view to always fill the `private` field.